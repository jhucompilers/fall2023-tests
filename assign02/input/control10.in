var i;
var n;

n = 11;

i = 0;
while (i < n) {
  var j;
  j = 0;
  while (j < n) {
    if (j == n-1) {
      println(8);
    } else {
      if ((i == 0 || j == 0) || i == n-1) {
        print(8);
      } else {
        if (j == i || n - j - 1 == i) {
          print(8);
        } else {
          print(1);
        }
      }
    }

    j = j + 1;
  }

  i = i + 1;
}

1;
