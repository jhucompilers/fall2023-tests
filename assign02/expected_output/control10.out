88888888888
88111111188
81811111818
81181118118
81118181118
81111811118
81118181118
81181118118
81811111818
88111111188
88888888888
Result: 1
